test/commands/register.test.ts

```diff
@@ -410,4 +410,136 @@ describe('register', function () {
       expect(compose.firstCall.args[0][5]).to.deep.equal("*.args.NODE_ENV=dev");
       expect(compose.firstCall.args[0][7]).to.deep.equal('*.args.SSH_PUB_KEY="abc==\ntest.architect.io"');
     });
+
+  mockArchitectAuth
+    .nock(MOCK_REGISTRY_HOST, api => api
+      .persist()
+      .head(/.*/)
+      .reply(200, '', { 'docker-content-digest': 'some-digest' })
+    )
+    .nock(MOCK_API_HOST, api => api
+      .persist()
+      .get(/\/accounts\/examples/)
+      .reply(200, mock_account_response)
+    )
+    .nock(MOCK_API_HOST, api => api
+      .persist()
+      .post(/\/accounts\/.*\/components/)
+      .reply(200, {})
+    )
+    .stdout({ print })
+    .stderr({ print })
+    .command(['register', '-a', 'examples', 'examples/react-app/architect.yml', 'examples/gcp-pubsub/pubsub/architect.yml'])
+    .it('register multiple apps at the same time with no tagged versions', ctx => {
+      expect(ctx.stderr).to.contain('Registering component react-app:latest with Architect Cloud...... done\n');
+      expect(ctx.stderr).to.contain('Registering component examples/gcp-pubsub:latest with Architect Cloud...... done\n');
+      expect(ctx.stdout).to.contain('Successfully registered component');
+    });
+
+  mockArchitectAuth
+    .nock(MOCK_REGISTRY_HOST, api => api
+      .persist()
+      .head(/.*/)
+      .reply(200, '', { 'docker-content-digest': 'some-digest' })
+    )
+    .nock(MOCK_API_HOST, api => api
+      .persist()
+      .get(/\/accounts\/examples/)
+      .reply(200, mock_account_response)
+    )
+    .nock(MOCK_API_HOST, api => api
+      .persist()
+      .post(/\/accounts\/.*\/components/)
+      .reply(200, {})
+    )
+    .stdout({ print })
+    .stderr({ print })
+    .command(['register', '-t', '1.0.0', '-a', 'examples', 'examples/react-app/architect.yml', 'examples/gcp-pubsub/pubsub/architect.yml'])
```

Comment:
I think that this is the most intuitive way to still be able to use the `-t` flag in combination with multiple components being registered, though I do think that we should mention how this works in the `-t` flag description

Reply:
Added description to the -t flag description